molo/core/tasks.py

```diff
@@ -38,8 +38,6 @@ def rotate_or_promote_content(day=None):
     if main and index:
         rotate_latest(main_lang, index, main, site_settings, day)
         rotate_featured_in_homepage(main_lang, day)
-        demote_articles()
-        promote_articles()
 
 
 def demote_articles():
```

Comment:
This should be decorated with `@task` and added to the celery schedule in the settings files